packages/kas/tsconfig-build.json

```diff
@@ -1,9 +1,8 @@
 {
-    "exclude": ["dist", "**/*.test.*", "**/*.testdata.*", "**/*.stories.*", "**/*.cypress.*"],
```

Comment:
It looks like we can provide the `exclude` param in the base tsconfig (`tsconfig-shared.json`), which simplifies these per-project `tsconfig-build.json` files somewhat.